alisa.go

```diff
@@ -65,20 +69,188 @@ func (ali *alisa) createTask(code string) (string, error) {
 	params["ExecTarget"] = ali.envs["ALISA_TASK_EXEC_TARGET"]
 	envBuf, _ := json.Marshal(ali.envs)
 	params["Envs"] = string(envBuf)
-	rsp, err := ali.pop.request(params, ali.popURL, ali.popSecret)
+
+	res, err := ali.requetAndParseResponse(params)
+	if err != nil {
+		return "", -1, err
+	}
+	var val alisaTaskMeta
+	if err = json.Unmarshal(*res, &val); err != nil {
+		return "", -1, err
+	}
+	return val.TaskID, val.Status, nil
+}
+
+// getStatus: returns the task status of taskID
+func (ali *alisa) getStatus(taskID string) (int, error) {
+	params := baseParams(ali.popID)
+	params["Action"] = "GetAlisaTask"
+	params["AlisaTaskId"] = taskID
+
+	res, err := ali.requetAndParseResponse(params)
+	if err != nil {
+		return -1, err
+	}
+	var val alisaTaskStatus
+	if err = json.Unmarshal(*res, &val); err != nil {
+		return -1, err
+	}
+	return val.Status, nil
+}
+
+// completed: check if the status is completed
+func (ali *alisa) completed(status int) bool {
+	return status == 3 || status == 4 || status == 6 || status == 8 || status == 9
```

Comment:
For better readability, please comment on the meaning of status code. Or change the number to `return status == statusSuccess ...`.